src/policy/fees.h

```diff
@@ -166,6 +166,7 @@ class CBlockPolicyEstimator
      * The MIN_BUCKET_FEERATE and MAX_BUCKET_FEERATE should just be set to
      * the lowest and highest reasonable feerate we might ever want to track.
      */
+    static constexpr double MIN_BUCKET_FEERATE = 4;
```

Comment:
The PR OP says "8 satoshi". I don't know if this conflicts with the concept ACKs on the PR.

Reply:
Right, if I remember correctly, the original patchset didn't preserve the estimate data, but did have a lower bound of exactly 8sat/kvb. The patchset now does preserve the old estimates, but as a result only approximates the lower bound -- choose 4 gives an error of 0.8% whereas 8 gave an error of 4.7%. Updated the PR description.